File: app/src/main/java/com/artmurka/artmurkaapp/Model/Pojo/ItemList/AboutGoods/DefPhoto.java

package com.artmurka.artmurkaapp.Model.Pojo.ItemList.AboutGoods;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class DefPhoto {

    @SerializedName("small")
    @Expose
    private String small;
    @SerializedName("photo")
    @Expose
    private String photo;
    @SerializedName("thumb")
    @Expose
    private String thumb;

    public String getSmall() {
        return small;
    }

    public void setSmall(String small) {
        this.small = small;
    }

    public String getPhoto() {
        return photo;
    }

    public void setPhoto(String photo) {
        this.photo = photo;
    }

    public String getThumb() {
        return thumb;
    }

    public void setThumb(String thumb) {
        this.thumb = thumb;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/aboutgoods/DefPhoto.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.aboutgoods

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class DefPhoto {

    @SerializedName("small")
    @Expose
    var small: String? = null
    @SerializedName("photo")
    @Expose
    var photo: String? = null
    @SerializedName("thumb")
    @Expose
    var thumb: String? = null

}
